client/post-editor/editor-media-advanced/index.jsx

```diff
@@ -2,42 +2,62 @@
  * External depencencies
  */
 import React, { PropTypes } from 'react';
-import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
 /**
  * Internal dependencies
  */
-import { toggleEditorMediaAdvanced } from 'state/ui/editor/media/actions';
+import { setEditorMediaEditItem } from 'state/ui/editor/media/actions';
 import Dialog from 'components/dialog';
 
-function EditorMediaAdvanced( { visible, toggleVisible } ) {
-	return (
-		<Dialog isVisible={ visible } onClose={ toggleVisible }>
-			WIP
-		</Dialog>
-	);
-}
+const EditorMediaAdvanced = React.createClass( {
+	propTypes: {
+		item: PropTypes.object,
+		resetEditItem: PropTypes.func,
+		insertMedia: PropTypes.func
+	},
+
+	getDefaultProps() {
+		return {
+			resetEditItem: () => {},
+			insertMedia: () => {}
+		};
+	},
 
-EditorMediaAdvanced.propTypes = {
-	visible: PropTypes.bool,
-	toggleVisible: PropTypes.func
-};
+	getButtons() {
+		return [
+			{
+				action: 'confirm',
+				label: this.translate( 'Save' ),
+				isPrimary: true,
+				onClick: () => this.props.insertMedia( 'Updated' )
+			}
+		];
+	},
+
+	render() {
+		const { item, resetEditItem } = this.props;
 
-EditorMediaAdvanced.defaultProps = {
-	visible: false,
-	toggleVisible: () => {}
-};
+		return (
+			<Dialog
+				isVisible={ !! item }
+				buttons={ this.getButtons() }
+				onClose={ resetEditItem }>
+				ID { item && item.media.ID }
```

Comment:
>maybe it's worth a translation

This is a WIP mostly to demonstrate that the parsing is working as expected. This will eventually be replaced with a more full-featured form (see #307).